graphql_compiler/__init__.py

```diff
@@ -218,3 +219,52 @@ def graphql_to_redisgraph_cypher(schema, graphql_query, parameters, type_equival
         schema, graphql_query, type_equivalence_hints=type_equivalence_hints)
     return compilation_result._replace(
         query=insert_arguments_into_query(compilation_result, parameters))
+
+
+def get_sqlalchemy_schema_info(
+    tables, sql_edge_descriptors, junction_tables, dialect, class_to_field_type_overrides=None
+):
+    """Return a SQLAlchemyInfo from the metadata.
+
+    Args:
+        tables: dict, str -> SQLAlchemy Table, mapping every GraphQL object in the schema to a
```

Comment:
Which graphql schema are you talking about in `the schema`?

Reply:
The schema that is about to be constructed.